e2e/app/geth/config.go

```diff
@@ -106,7 +106,8 @@ func MakeGethConfig(conf Config) FullConfig {
 
 	// Set the bootnodes and trusted nodes.
 	cfg.Node.UserIdent = conf.Moniker
-	cfg.Node.P2P.DiscoveryV4 = true // TODO(corver): Switch to v5.
+	cfg.Node.P2P.DiscoveryV4 = true
+	cfg.Node.P2P.DiscoveryV5 = true
```

Comment:
Is enabling both safe / is this necessary for 1.14.11?

Reply:
Opened this issue https://github.com/omni-network/omni/issues/2069